### DISCUSSION TITLE
Slingplayer install issues

### DISCUSSION THREAD

**Discussion by mesaka123** [GerryDazoo/Slinger#654] [General]
Hi all,  I was clearing out my basement and found my M1 gathering dust.  Looked through this site aiming to resurrect it but I am stuck with the most annoying issue - I install slingplayer on my Win11 machine and it seems to install properly but when I launch it all I get is the splash screen.  Has anyone any idea how to get it running so I can extract my password?
thanks in advance!

**Comment by GerryDazoo** [comment]
What version of SlingPlayer did you install? You need 5.0.0.83 for the M1.
Did you block access to the Sling servers on your PC? 
That said, You can't recover the password using SlingPlayer. 
There's other threads here on how to recover the password from the Sling servers.
You can factory reset the box to reset the admin password to "admin" but this will wipe out any IR remote configuration.

**Comment by mesaka123** [comment]
Thanks for responding - yes I downloaded SlingplayerDesktop-5.0.0.83.  I hoped to follow the instructions [here ](https://github.com/GerryDazoo/Slinger/discussions/374) to use it to get my password.  I do have a couple of custom configs in there but I will use the reset option if I have to.

**Comment by mesaka123** [comment]
Thanks for responding - yes I downloaded SlingplayerDesktop-5.0.0.83.  I hoped to follow the instructions [here ](https://github.com/GerryDazoo/Slinger/discussions/374) to use it to get my password.  I do have a couple of custom configs in there but I will use the reset option if I have to.
